app/screens/settings/display_settings/display_settings.js

```diff
@@ -72,12 +73,30 @@ export default class DisplaySettings extends PureComponent {
         });
     });
 
+    goToThemeSettings = preventDoubleTap(() => {
+        const {navigator, theme} = this.props;
+        const {intl} = this.context;
+
+        navigator.push({
+            screen: 'ThemeSettings',
+            title: intl.formatMessage({id: 'mobile.advanced_settings.theme', defaultMessage: 'Theme'}),
```

Comment:
id should be something like `mobile.display_settings.theme`

Reply:
Was placed there to mirror "mobile.advanced_settings.clockDisplay" in the same section. Will update.